### PR TITLE
chore: better logging for devnet upgrade protocol test

### PR DESCRIPTION
**Pull request by ogabrielides**
## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [ ] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs [hunk 1]
```diff
@@ -42,13 +48,37 @@ impl<C> Platform<C> {
...
+        );
+
         if !versions_passing_threshold.is_empty() {
             // same as equals 1
             let next_version = versions_passing_threshold.remove(0);
 
             // TODO: We stored next version here previously.
             //  It was never used so we can temporary remove it from here and move it to Epoch trees in upcoming PR
 
+                tracing::info!(
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs:73]: I would move it there https://github.com/dashpay/platform/blob/a12abe641322e503b1cce4195b34783750a77938/packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/upgrade_protocol_version/v0/mod.rs#L68-L67

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/upgrade_protocol_version/v0/mod.rs [hunk 2]
```diff
@@ -66,6 +66,13 @@ impl<C> Platform<C> {
                 self.check_for_desired_protocol_upgrade(hpmn_list_len, platform_version)?;
 
             if let Some(protocol_version) = next_epoch_protocol_version {
+                tracing::trace!(
+                    current_epoch_index = epoch_info.current_epoch_index(),
+                    next_protocol_version = protocol_version,
+                    "Next epoch protocol version: current epoch_index: {}, next_protocol_version: {}",
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/upgrade_protocol_version/v0/mod.rs:72]: next protocol version set to {protocol_version}

File: packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs [hunk 3]
```diff
@@ -42,13 +48,21 @@ impl<C> Platform<C> {
             ));
         }
 
+        tracing::trace!(
+            total_hpmns = total_hpmns,
+            required_upgraded_hpmns = required_upgraded_hpmns,
+            current_votes = votes_str,
+            versions_passing_threshold = versions_passing_threshold.len(),
+            "Check protocol upgrade: check_for_desired_protocol_upgrade: total_hpmns: {}, required_upgraded_hpmns: {}, current_votes: {}, versions_passing_threshold: {}",
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs:56]: "Protocol version voting is finished. {} versions passing the threshold: {:?}", versions_passing_threshold.len(), versions_passing_threshold

**Review by shumkov — CHANGES_REQUESTED** [review]

File: packages/rs-drive-abci/src/abci/handler/prepare_proposal.rs [hunk 4]
```diff
@@ -207,7 +210,9 @@ where
         valid_tx_count,
         delayed_tx_count,
         failed_tx_count,
-        "Prepared proposal with {} transition{} for height: {}, round: {} in {} ms",
+        storage_fees,
+        processing_fees,
+        "Prepared proposal with {} transition{} for height: {}, round: {}, storage_fees: {}, processing_fees: {} in {} ms",
```
[the remark this answers is not in the document]
Comment (ogabrielides) [reply packages/rs-drive-abci/src/abci/handler/prepare_proposal.rs:215]: I didn't understand this suggestion. Can you explain please @shumkov ?

[the remark this answers is not in the document]
Comment (shumkov) [reply packages/rs-drive-abci/src/abci/handler/prepare_proposal.rs:215]: Tracing event accept params and message. I suggested to do out include fees into message and keep only in params to do mot overcomplicate it

File: packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs [hunk 5]
```diff
@@ -42,13 +48,19 @@ impl<C> Platform<C> {
             ));
         }
 
+        tracing::trace!(
+            total_hpmns = total_hpmns,
+            required_upgraded_hpmns = required_upgraded_hpmns,
+            current_votes = votes_str,
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs:54]: all_votes = ?protocol_versions_counter.global_cache

File: packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs [hunk 6]
```diff
@@ -30,25 +30,33 @@ impl<C> Platform<C> {
...
+        tracing::trace!(
+            total_hpmns,
+            required_upgraded_hpmns,
+            all_votes = ?protocol_versions_counter.global_cache,
+            versions_passing_threshold = versions_passing_threshold.len(),
+            "Protocol version voting is finished. {} versions passing the threshold: {:?}",
+            versions_passing_threshold.len(),
+            versions_passing_threshold
+        );
+        drop(protocol_versions_counter);
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs:55]: Same here

File: packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs [hunk 7]
```diff
@@ -30,25 +30,33 @@ impl<C> Platform<C> {
         // if we are at an epoch change, check to see if over 75% of blocks of previous epoch
         // were on the future version
         let protocol_versions_counter = self.drive.cache.protocol_versions_counter.read();
+
         let mut versions_passing_threshold =
             protocol_versions_counter.versions_passing_threshold(required_upgraded_hpmns);
-        drop(protocol_versions_counter);
 
         if versions_passing_threshold.len() > 1 {
+            drop(protocol_versions_counter);
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs:38]: This variable will be dropped on the next line. I don't think you need to do explicitly.

**Review by shumkov — CHANGES_REQUESTED** [review]

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**